src/main/java/org/fundacionjala/coding/jorge/SortTwisted37.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.coding.jorge;
+
+import java.util.Arrays;
+
+/**
+ *
+ */
+public final class SortTwisted37 {
+    /**
+     *
+     */
+    private static final char NUMBER_7 = '7';
+    private static final char A = 'a';
+    private static final char NUMBER_3 = '3';
+
+    /**
+     *
+     */
+    private SortTwisted37() {
+
+    }
+
+    /**
+     * @param array to be sorted
+     * @return sorted array
+     */
+    public static Integer[] sortTwisted37(final Integer[] array) {
+        return Arrays.stream(array.clone())
+                .map(i -> twist(i))
+                .sorted((a, b) -> a.compareTo(b))
+                .map(i -> twist(i))
+                .toArray(Integer[]::new);
```

Comment:
This may be refactored like this.
```java
        return Arrays.stream(array)
                .map(SortTwisted37::twist)
                .sorted()
                .map(SortTwisted37::twist)
                .toArray(Integer[]::new);
```